src/main/java/build/buildfarm/common/redis/RedisPriorityQueue.java

```diff
@@ -277,13 +310,29 @@ private void visit(String queueName, StringVisitor visitor) {
     } while (entries.size() == listPageSize);
   }
 
+  private String getPushLuaScript() {
+    return String.join(
+        "\n",
+        "local setName = KEYS[1]",
+        "local maxNodeQueueDepth = ARGV[1]",
+        "local priority = ARGV[2]",
+        "local value = ARGV[3]",
+        "local queueSize = redis.call('ZCARD', setName)",
+        "if queueSize < maxNodeQueueDepth then",
+        "  redis.call('ZADD', setName, priority, value)",
+        "  return 't'",
+        "end",
+        "return 'f'");
+  }
+
```

Comment:
We now support multiline strings (text blocks) in java, good opportunity to switch this up as a constant: https://docs.oracle.com/en/java/javase/15/text-blocks/index.html